lipsweb/models.py

```diff
@@ -0,0 +1,18 @@
+from cms.models.pluginmodel import CMSPlugin
+
+from django.db import models
+
+
+class Submission(CMSPlugin):
+    content = models.TextField(default='')
+    upvoteCount = models.PositiveIntegerField(default=0)
+
+    def countSubmission(self):
+        return len(self.content)
+
+    def countUpvote(self):
+        return len(self.upvoteCount)
+
+
+class AboutLips(CMSPlugin):
```

Comment:
I'm assuming you tested this and it works, so I won't ask you to change it, but it seems like you may have done more work than is necessary. I think the plugins are intended for modular/reusable components.  
I think that this about lips page is just a bunch of text right?
I'm wondering if you could have just used the existing 'text' component from the CMS, and then used some custom CSS to format it however you wanted on that page. 

So don't bother changing this but if we have to make any more static pages, let's see what the easiest way to do it might be.  Maybe a Django CMS template? 
https://www.django-cms.org/en/blog/2016/05/11/frontend-development-with-django-cms-best-practices/